synapse/lib/spooled.py

```diff
@@ -166,27 +166,27 @@ async def set(self, key, val):
             self.len = len(self.realdict)
             self.realdict.clear()
 
-    def pop(self, key, defv=None):
+    def pop(self, key, defv=None, use_list=False):
```

Comment:
```suggestion
    def pop(self, key, defv=None, *, use_list=False):
```
Do we want to make `use_list` here a keyword-only argument? Its not uncommon for folks to use `default` dictionary get style operations without a keyword, but we probably don't want to encourage this object-mutation being allowed without a keyword.

Reply:
Yep, great idea!